content/docs/atom-guide/look-dev/materials/wrinkle-masks.md

```diff
@@ -0,0 +1,60 @@
+
+## Mask Blending
+It is possible for multiple overlapping morph targets to be active at once. In this case, the contribution of each mask will be summed.
+
+As an example, let's say that for the left_eyebrow_raised morph target, the portion of the forehead directly above the left eyebrow is fully wrinkled using the first wrinkle layer. This means that portion of the mask will be bright red, indicating that the first wrinkle layer should be used for the normal. For the portion of the mask that covers the center of the forehead, the mask might be a darker, mid-range red that indicates that the middle of the forehead is only halfway to fully wrinkled when the left eyebrow is raised.
+
+The right_eybrow_raised mask might be a mirror or nearly-mirror image of the left mask, with the center of the forehead also halfway wrinkled. But if both the left eyebrow and right eyebrow are fully raised at the same time, the overlapping values in the center of the forehead from the two masks will be added together, resulting in the center of the forehead being fully wrinkled using the first wrinkle layer.
+
+When the values from multiple masks are combined, they will be clamped in the 0-1 range, so they will never exceed 1 for any given layer.
+
+## Corrective Shapes
+It's not clear if corrective shapes will need their own mask. If they are just being used to smooth out the transition between two other shapes, the combined mask values from the other shapes may be enough. Since the blend masks are only additive, a corrective shape will not be able to reduce the influence of a wrinkle layer, only add to it.
+
+## Base Color/Normal Blending
+Once the masks for any active morph targets have been combined, the resulting rgba value will be used to blend the base color and normal maps from the different wrinkle layers in the skin material. These will be blended the same way they would have been blended if a static model with painted vertex colors was used.
+
+MultiLayerPBR supports combining vertex colors with a mask for blending between layers. The Skin shader does not. Instead, if any masks exist, the resulting values from the masks will be used for blending and the vertex colors will be ignored (across the entire mesh, not just the portion impacted by the morph target).
+
+The value in the red-channel is used to lerp between the base and wrinkle layer 1. The result is then lerped with wrinkle layer 2 using the value from the green-channel. Then 3 then 4. This results in an order-dependent value, so changing the order of the wrinkle layers impacts the visual result.
```

Comment:
How does a user get to the state where the active morph targets are combined? I think that question should be addressed somewhere in the doc.